optlang/tests/test_cplex_interface.py

```diff
@@ -497,6 +499,98 @@ def test_qp_non_convex(self):
             model.optimize()
             self.assertAlmostEqual(model.objective.value, 2441.999999971)
 
+    class InfeasibleMIPTestCase(unittest.TestCase):
```

Comment:
I agree that it's good to test this as well, and I just meant that these tests could simply be added to the other test case.